### PR TITLE
docs(modeler): stub pages, point to new docs

### PR DESCRIPTION
**Pull request by akeller**
Completing Modeler docs content migration to docs.camunda.io. 

Related to [CAM-14335](https://jira.camunda.com/browse/CAM-14335).

### CODE REVIEW

File: content/introduction/licenses.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The Camunda Modeler is a desktop modeling application.
 
 ## Third-Party Libraries
 
-[Third-Party libraries](https://docs.camunda.io/docs/reference/dependencies/) for Camunda Modeler have moved to docs.camunda.io.
+[Third-Party libraries](https://docs.camunda.io/docs/reference/dependencies/) for Camunda Modeler have moved to [docs.camunda.io](https://docs.camunda.io/).
```
Comment (MaxTru) [inline content/introduction/licenses.md:48]: Why not deeplink to https://docs.camunda.io/docs/reference/dependencies/?

File: content/modeler/_index.md [hunk 2]
```diff
@@ -14,8 +14,4 @@ menu:
 
 ---
 
-Camunda Modeler is a desktop application for modeling BPMN and DMN. It allows you to model files located directly on your local file system.
-
-{{< img src="img/camunda-modeler.png" title="Camunda Modeler" >}}
-
-This is a guide to get you started with modeling BPMN diagrams and DMN decision tables. Make sure to [Install the Camunda Modeler]({{< ref "/installation/camunda-modeler.md" >}}) first.
+This content has moved to (docs.camunda.io)[https://docs.camunda.io/] and is available [here](https://docs.camunda.io/docs/components/modeler/about-modeler/) for all future versions.
```
Comment (MaxTru) [inline content/modeler/_index.md:17]:
```suggestion
This content has moved to [docs.camunda.io](https://docs.camunda.io/) and is available [here](https://docs.camunda.io/docs/components/modeler/about-modeler/) for all future versions.
```

**Review by MaxTru — CHANGES_REQUESTED** [review]
In multiple cases you put
```
(docs.camunda.io)[https://docs.camunda.io/] 
```

but you need to put
```
[docs.camunda.io]/https://docs.camunda.io/])
```

File: content/modeler/forms.md [hunk 3]
```diff
@@ -12,39 +12,4 @@ menu:
...
-{{< img src="img/form-properties-panel.png" title="Camunda Form Properties Panel" >}}
-
-## Save your Form
-
-To save your state of work, click the **File > Save File As...** button in the top-level menu. Then select a location on your file system to store the form as `.form` file. You can load that file again by clicking **File > Open File...**.
-
-## Connect your Form to a BPMN diagram
-
-You can connect your Camunda Form to a User Task or Start Event, so to implement a task form in your application. Refer to the [User Task Forms guide]({{< ref "/user-guide/task-forms/_index.md#camunda-forms" >}}) to learn how.
+This content has moved to (docs.camunda.io)[https://docs.camunda.io/] and is available [here](https://docs.camunda.io/docs/guides/utilizing-forms/) for all future versions.
```
Comment (MaxTru) [inline content/modeler/forms.md:15]: The Forms Reference in the Cloud docs in not yet ready for Platform. E.g., there is no link to [Camunda Forms Reference]({{< ref "/reference/forms/camunda-forms/_index.md#configuration" >}}) which explains how to implement the linking. Will this be added?

File: content/introduction/licenses.md (hunk 1 above)
Comment (akeller) [reply content/introduction/licenses.md:48]: For both links? I did on the first one but was adding the additional link to the overall docs.

File: content/modeler/forms.md (hunk 3 above)
Comment (akeller) [reply content/modeler/forms.md:15]: I'm not sure I understand. Did this content not move to docs.camunda.io? Or am I referencing the wrong URL? If there is something that needs to be added to documentation content on docs.camunda.io can you create a PR or issue for that?

Comment (MaxTru) [reply content/modeler/forms.md]: Sure, => https://github.com/camunda-cloud/camunda-cloud-documentation/issues/613

**Review by MaxTru — APPROVED** [review]

**Review by MaxTru — APPROVED** [review]

**Comment by akeller** [comment]
I'm having trouble signing the CLA. Also, I wasn't sure who is supposed to merge this. Is it me? 😅

**Comment by MaxTru** [comment]
> I'm having trouble signing the CLA. Also, I wasn't sure who is supposed to merge this. Is it me? sweat_smile

I think this should auto-merge once you signed the CLA (IT could help). Or you just manually merge? :-) :bomb:

**Pull request merged**